src/app/helpers/sortBy.ts

```diff
@@ -0,0 +1,3 @@
+export const sortBy = (key: string) => {
+  return (a: any, b: any) => (a[key] > b[key] ? 1 : b[key] > a[key] ? -1 : 0);
```

Comment:
if parameters are going to be used as objects with keys lets type them as Object instead of any to avoid problems when calling this function with wrong parameters